src/patchelf.cc

```diff
@@ -2242,6 +2489,25 @@ int mainWrapped(int argc, char * * argv)
         else if (arg == "--add-debug-tag") {
             addDebugTag = true;
         }
+        else if (arg == "--rename-dynamic-symbols") {
+            renameDynamicSymbols = true;
+            if (++i == argc) error("missing argument");
+
+            std::ifstream infile(argv[i]);
```

Comment:
Should this default to stdin if `-` is passed?

Reply:
Not so sure anymore since we have multiple file arguments and it can be difficult to keep it consistent...